charts/datadog/values.yaml

```diff
@@ -313,7 +313,7 @@ datadog:
     enableOOMKill: false
 
     # datadog.systemProbe.collectDNSStats -- Enable DNS stat collection
-    collectDNSStats: false
+    collectDNSStats: true
```

Comment:
does it means that SystemProbe is required by default?

Reply:
It is the latter. DNS stats will get collected only when `networkMonitoring.enable` is also `true`

Reply:
Thanks for the answer. 
If you can rebase you pull request, we will be able to merge it.

Reply:
Done